src/main.rs

```diff
@@ -180,6 +187,11 @@ async fn main() {
         web::web_main(port).await;
     });
 
+    if autolaunch_ui {
+        open::that(Path::new(&format!("http://localhost:{}", port)))
+        .expect("Failed to open web browser");
```

Comment:
Maybe this should just trace an error, not being able to open the browser automatically shouldn't prohibit the app from continuing, even if the user wanted it to launch the browser.